packages/altive_lints/lib/src/lints/avoid_hardcoded_japanese.dart

```diff
@@ -40,6 +40,12 @@ class AvoidHardcodedJapanese extends DartLintRule {
     ErrorReporter reporter,
     CustomLintContext context,
   ) {
+    if (resolver.source.uri.toString().contains('/test/')) {
+      return;
+    }
+    // Although redundant, _test.dart is temporarily excluded here
+    // because the warning message appears in the _test.dart file
+    // in the lint_test directory.
```

Comment:
![badge](https://img.shields.io/badge/review-imo-important)
It is not necessary to write in the product code of the test thing.
Please respond to changes in requirements in the product code on the load and test side.

Reply:
Indeed, the logic targeting test code and test directories seems unnecessary, so we will remove this one.

Reply:
It would be better to leave the '_test.dart' decision in place for backward compatibility, just in case.

Reply:
Thank you very much.

In the meantime, I have taken the form of leaving `_test.dart` and deleted the unnecessary comment content.

Change : https://github.com/altive/altive_lints/pull/70/commits/5f6dc0bc3337e5f654fe23b63672637017578058